src/components/UI/Display/MAPI/permissions/PermissionsUseCases/UseCaseStatus.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Text } from 'grommet';
+import React from 'react';
+import styled from 'styled-components';
+import theme from 'styles/theme';
+import ScreenReaderText from 'UI/Display/ScreenReaderText';
+
+const Icon = styled(Text)`
+  font-size: 20px;
+`;
+
+interface IUseCaseStatusProps {
+  value?: boolean;
+  useCaseName: string;
+  organizationName?: string;
+}
+
+const UseCaseStatus: React.FC<IUseCaseStatusProps> = ({
+  value,
+  useCaseName,
+  organizationName,
+}) => {
+  return (
+    <Box
+      direction='row'
+      align='center'
```

Comment:
<img width="80" alt="Screen Shot 2022-05-05 at 15 59 39" src="https://user-images.githubusercontent.com/62935115/166939660-65232c9e-815b-4bd3-9db9-55a0f03c6b4d.png">

Could we align the statuses to the start so the "Yes" and "No" statuses are in line?

Reply:
Wait, no. I'd rather use `Partial` than `Mixed`.

Reply:
Here's the version with fixes. `Yes` and `No` statuses aligned to the left of each other. `Partial` status, since it's much longer and only appears in a collapsed view, has center alignment.

<img width="420" alt="Screenshot 2022-05-06 at 10 28 21" src="https://user-images.githubusercontent.com/445309/167087424-06a4b2c8-9bc0-4c1e-9f16-d2cdadb9eb39.png">